module-2/backend/week_one.md

```diff
@@ -5,30 +5,80 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. List the five common HTTP verbs and what the purpose is of each verb.
-2. What is Sinatra?
+  POST - create
+  GET - read
+  PUT - update/replace
+  PATCH - update/modify
+  DELETE - delete
+  
+2. What is Sinatra? 
+  a framework for Ruby
+
 4. What is MVC?
+  the common folder structure for web applications consisting of model-the info tracker/where 'work' takes place, views-what users see, controller-the delegater/sends you to the correct location
+  
 5. Why do we follow conventions when creating our actions/path names in our Sinatra routes?
+  to make our code easy to read/follow for other developers
+  
 6. What types of variables are accessible in our view templates without explicitly passing them?
+  HTML elements; all variables/data are sent from the controller
+  
 7. Given the following block of code, how would I pass an instance variable `count` with a value of `1` to my `index.erb` template?
   
   ```ruby
   get '/horses' do
+    @count = 1
+    @name = Mr. Ed
     erb :index
   end
   ```
 
 8. In the same code block, how would I pass a local variable `name` with a value of `Mr. Ed` to the view?
+  ^see above^
+  
 9. What's the purpose of ERB?
+  Embedded Ruby files, enables you to embed Ruby code/methods into HTML output for web pages
+  
 10. Why do I need a development AND test database?
+  because they set the environment (development, test, or production)

```

Comment:
This is a little bit backwards. We set the environment variables to tell our application which database to use. We need separate databases because each serves a different purpose. Development is for us to have some local representation of the database available for us to browse. Test is for our tests to use without changing development or production. Production is for the live site with actual users.